build.gradle

```
@@ -507,6 +507,12 @@ ext {
                 group  : 'com.oracle.database.jdbc',
                 name   : 'ojdbc8'
             ],
+            // Deprecated: Remove in the next major version
```

```suggestion
            // TODO Deprecated: Remove in the next major version
```

This way at least appears in Intellij's TODO window.

---

Done. Good catch!